pipeline_ukbiobank/cli/compute_stats.py

```diff
@@ -143,25 +142,25 @@ def compute_predictors_statistic(df):
         stats[predictor]['med'] = np.median(df[predictor])
         stats[predictor]['mean'] = np.mean(df[predictor])
     # Computes male vs female ratio
-    stats['Sex'] = {}
-    stats['Sex']['%_M'] = 100*(np.count_nonzero(df['Sex']) / len(df['Sex']))
-    stats['Sex']['%_F'] = 100 - stats['Sex']['%_M']
+    stats['Gender'] = {}
```

Comment:
Change back to 'Sex' as per my comment https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/74#issuecomment-850544868.  I'm curious about what made you change it from Sex to Gender?

Reply:
I honestly don't remember 😬 I don't think it was that well thought out, I juste checked in UK Biobank they call it 'Sex' too, I'll change it back!

Reply:
Done in https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/74/commits/654fd510ebc4e7d48ffbce6613b55ab8eb18ecbe